garrysmod/lua/vgui/dcolormixer.lua

```diff
@@ -340,7 +336,11 @@ function PANEL:DoConVarThink( convar )
 
 	if ( !convar ) then return end
 
-	local fValue = GetConVarNumber( convar )
+	local cVar = GetConVar(convar)

```

Comment:
Since it gets the convar again GetConVarString is suitable here

Reply:
Yes but GetConVarString is deprecated, as stated by the wiki. https://wiki.facepunch.com/gmod/Global.GetConVarString

Reply:
The deprecation warning will probably be removed.

Reply:
If that's the case then this PR is utterly pointless lol.